tap_stripe/__init__.py

```diff
@@ -894,6 +894,10 @@ def sync():
 def main():
     # Parse command line arguments
     args = utils.parse_args(REQUIRED_CONFIG_KEYS)
+    # set the config and state in prior to check te authentication in the discovery mode itself.
```

Comment:
Spelling error in Comment. Please correct it

Reply:
Corrected the mistake